schema.sql

```diff
@@ -7,4 +7,6 @@ CREATE TABLE
         escape_attempts INT,
         neutered BOOLEAN,
         weight_kg DECIMAL
-    );
\ No newline at end of file
+    );
+
+ALTER TABLE animals ADD species TEXT;
```

Comment:
- [ ] I would like to recommend you change the species datatype from **TEXT** to **varchar** because This data type is used to store characters of limited length like `species`. You could have something like the example below :arrow_down: 
``` 
ALTER TABLE animals ADD species varchar(255);

```